src/applications/rated-disabilities/components/OnThisPage.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+import { OnThisPageLink } from './OnThisPageLink';
+
+export default function OnThisPage() {
+  return (
+    <div className="usa-width-one-third">
+      <h2 className="vads-u-font-size--h3">On this page</h2>
```

Comment:
I don't think the "On this page" text is inside a header. The [web component](https://design.va.gov/storybook/?path=/docs/components-va-on-this-page--default) sets the text to 2rem, but it's not a header.

Reply:
I have an inquiry with DK (our accessibility guy) asking him about this. I would like to have replaced this whole thing with the `va-on-this-page` web component, but I have an open ticket with the design system team to verify an issue I am seeing with it. This work is just to transplant the code from the `RatedDisabilityView` component into a new component - I'm planning on revisiting this one soon